src/components/EventForm.js

```diff
@@ -177,10 +168,32 @@ class EventForm extends Component {
                   <Input
                     multiline
                     onChangeText={(change)=> {
-                      props.setFieldValue("description", change);
+                      props.setFieldValue("description.en", change);
+                    }}
+                    style={{ paddingTop: 15, paddingBottom: 15 }}
+                    value={props.values.description.en}
+                  />
+                </Item>
+                <Item>
+                  <Label>Description (ES)</Label>
```

Comment:
I wonder if there's a way we can encapsulate the idea of repeating the same form element multiple times for each language option in a component, like `<LocalizedInputs>` that takes the attribute it's affecting, like `description` in this case. Not something we need to do right now, but if we need to do a lot of this kind of thing, we should consider it later.

Reply:
Good idea!